app/javascript/react/store/fixedSessionsSelectors.ts

```diff
@@ -54,8 +63,30 @@ const selectFixedSessionsList = createSelector(
     )
 );
 
+const selectFixedSessionPointsBySessionId = (sessionId: number | null) =>
+  createSelector([selectFixedSessionsState], (fixedSessionState): Session[] => {
+    const fixedSessionByStreamId = fixedSessionState.sessions.find(
+      (session) => Number(session.id) === Number(sessionId)
+    );
+    const streams = fixedSessionByStreamId?.streams || {};
+
+    return [
+      {
+        id: fixedSessionByStreamId?.id || 0,
+        lastMeasurementValue: fixedSessionByStreamId?.lastMeasurementValue || 0,
+        point: {
+          lat: fixedSessionByStreamId?.latitude || 0,
+          lng: fixedSessionByStreamId?.longitude || 0,
+          streamId: streams[Object.keys(streams)[0]]?.id.toString() || "0",
```

Comment:
i would move  `streams[Object.keys(streams)[0]]` . Probably everywhere in this file, not only in yours changes

Reply:
done 🙌